refinery/ui/source/js/dashboard/controllers/data-sets-card-ctrl.js

```diff
@@ -133,6 +134,30 @@
       });
     }
 
+     /**
+     * @name openDataSetTransferModal
+     * @desc  Opens data set tranfer modal
```

Comment:
Typos: I'm seeing `tranfer` instead of `transfer` a couple of times now

Reply:
Ok. I realize I have an issue with typing transfer. Apologizes.